.prow-images

```diff
@@ -0,0 +1 @@
+prow/cmd/sinker
```

Comment:
something to consider later: we have a lot of "dot files" accumulating in the repo root, I think for things like "go version" it makes sense not to bury these, they will be edited somewhat often and are repo-wide.

for things like "prow images" I think we should consider placing them closer to the sources and not all in the repo root.